Arduino_package/hardware/libraries/SPI/examples/ST7735_TFT_LCD_basic/ST7735_TFT_LCD_basic.ino

```diff
@@ -0,0 +1,180 @@
+/*
+ * This sketch demonstrates how to use TFT LCD with ILI9314 api
```

Comment:
We can add some discerptions at top, to indicate you are the owner and have the copy right/license of this codes.